.ci/azure/linux_ngraph_onnx.yml

```diff
@@ -82,3 +82,86 @@ jobs:
       docker run --name openvino-onnx-ci-container --volume $(TMP_DIR)/model_zoo/onnx_model_zoo_$(ONNX_MODEL_ZOO_SHA):/root/.onnx/model_zoo/onnx_model_zoo --volume $(MODELS_DIR)/msft:/root/.onnx/model_zoo/MSFT openvino-onnx-ci-image /bin/bash -c "tox && tox -e zoo_models"
     displayName: 'Docker run'
 
+- job: nGraph_ONNX_Lin_Debug
```

Comment:
you can avoid code duplication, e.g. Matrix https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema

Reply:
Very very good point :)